src/main/scala/ru/org/codingteam/keter/game/objects/equipment/items/Weapon.scala

```diff
@@ -37,4 +36,14 @@ case class Knife(name: String,
 
   override val requires: Set[Capability] = Set(ManipulatorCapability)
 
+  override def damage: Int = _damage
+
+  /**
+   * Name of equipment item.
+   */
+  override def name: String = _name
+}
+
+object Knife {
+  def apply(name: String, damage: Int = 50) = new Knife(name, damage)
```

Comment:
Honestly, I dislike the approach taken. Now we'll be required to be very careful with declaring our item classes to not hit the same issue with any of them.

There's an alternate route with marking our items with identifiers (the same as we already use for actors and factions). 

@ttldtor, what do you think about this alternative? Would it be worse?

Reply:
I don't see any faction id: `case class Faction(name: String)`. Yep, now I agree with you. I'll add id to items.